web/src/runtime.ts

```diff
@@ -1477,6 +1489,41 @@ export class Instance implements Disposable {
     this.cacheMetadata = { ...this.cacheMetadata, ...(list["metadata"] as Record<string, any>) };
   }
 
+  /**
+   * Delete NDArray from cache url.
```

Comment:
Would be great if we could have more documentation on this function's behavior. e.g. "given `cacheUrl`, we search up items to delete based on `cacheUrl/ndarray-cache.json`", etc. 

One thing worth pointing out is that, the `ndarray-cache.json` and tokenizer files would still linger in the cache after this function. We only delete the shard files.